src/styles/blocks/catalog.scss

```
@@ -17,6 +17,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms;
```

add color property too
```suggestion
    transition: transform 300ms, color 300ms;
```